lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -711,13 +711,12 @@
       display: block;
       position: absolute;
       top: ($baseline/2);
-
-      @include left(-9999em);
-
+      left: -9999em;
```

Comment:
Forgive my late question, why not `display: none`?

`left: -9999em;` seems like a bad way to hide an element and could cause other sort of nasty bugs.


Check the todo below:


```
/* ***
     * Ideally we want to handle functionality with JS.
     * This functionality should eventually be moved into CS/JS, and out of here. */
    .has-hint:hover > .hint {
      @include left($baseline*10);
    }
```

If we're hardcoding `left` here. How it would play with `@include left($baseline*10);`?

In its current form it looks a bit oddly placed in Arabic, what's the placement in English look like in a screenshot?

<kbd>![image](https://github.com/user-attachments/assets/36cea351-66ae-4a66-aa57-b26388be24a0)</kbd>